src/schema/imported/manifest.json

```diff
@@ -241,17 +241,23 @@
                 },
                 "optional_permissions": {
                   "type": "array",
+                  "default": [],
                   "items": {
                     "allOf": [
                       {
                         "$ref": "#/types/OptionalPermissionOrOrigin"
                       },
                       {
                         "onError": "warn"
+                      }],
+                    "anyOf": [
+                      {},
+                      {
+                        "format": "deprecated"
                       }
                     ]
                   },
-                  "default": []
+                  "uniqueItems": true
```

Comment:
@aa-deet-eeya This isn't yet the same as it will be once we will import the schema, the properties have to be exactly in the order as in the diff I pasted in my previous comment and the `allOf` array formatted as the `anyof` (currently it is not).

Here is a diff that would apply these changes on top of yours:
```diff
diff --git a/src/schema/imported/manifest.json b/src/schema/imported/manifest.json
index 8a0a24e5..16356cc8 100644
--- a/src/schema/imported/manifest.json
+++ b/src/schema/imported/manifest.json
@@ -241,7 +241,6 @@
                 },
                 "optional_permissions": {
                   "type": "array",
-                  "default": [],
                   "items": {
                     "allOf": [
                       {
@@ -249,7 +248,8 @@
                       },
                       {
                         "onError": "warn"
-                      }],
+                      }
+                    ],
                     "anyOf": [
                       {},
                       {
@@ -257,6 +257,7 @@
                       }
                     ]
                   },
+                  "default": [],
                   "uniqueItems": true
                 },
                 "web_accessible_resources": {
```

(you could also just same this into a text file and then apply them using something like `patch -p1 < path/to/file.patch`).

Reply:
I fixed this formatting issue :)
Also, could you explain why the position of `"default": [],` needs to be in the same exact position?

Reply:
> I fixed this formatting issue :)
> Also, could you explain why the position of `"default": [],` needs to be in the same exact position?

@aa-deet-eeya Sure thing (and my apologies, my previous comments didn't describe that in detail):

as I did briefly mention in my previous comments about this, we do import the API schema files from Firefox beta builds in a semi-automated way and so to prevent the next import to include unrelated changes we have to make sure that the change applied to src/schema/imported/manifest.json is exactly as the one that our script would apply.

If you want to double-check that locally you can execute the following commands: 
```
$ ./scripts/download-import-tag FIREFOX_81_0b7_RELEASE
...
$ git diff src/schema/imported
...
```
(but take into account that it will download a zip file ~600mb big, and so it will take some time and network bandwidth).

The last version of this patch does apply the same kind of change that the script would apply based on what we have added to `src/schema/updates/manifest.json`, and so `git diff` should now show you only some small new changes to the `src/schema/imported/chrome_settings_overrides.json` schema file and no additional changes to `src/schema/imported/manifest.json` (as we want and we expected after your last fix on the change applied in this PR to that file).